src/coreclr/vm/marshalnative.cpp

```diff
@@ -69,6 +69,22 @@ VOID QCALLTYPE MarshalNative::Prelink(MethodDesc * pMD)
     END_QCALL;
 }
 
+// IsComSupported
+// Reads the config file from native to ensure the runtime is in a consistent state
```

Comment:
> config file

Nit: the runtime is not reading from a config file. (The host reads the config file and passes properties to the runtime.)